enterprise/backend/test/metabase_enterprise/scim/v2/api_test.clj

```diff
@@ -142,8 +165,9 @@
                           :active true}
                 response (scim-client :post 201 "ee/scim/v2/Users" new-user)]
             (is (malli= scim-api/SCIMUser response))
-            (is (=? (select-keys user [:email :first_name :last_name :is_active])
-                    (t2/select-one [:model/User :email :first_name :last_name :is_active]
+            (is (=? (assoc (select-keys user [:email :first_name :last_name :is_active])
```

Comment:
nit: Could this be `=` instead of `=?`?

Reply:
it's subtle, but fetching a user also hydrates the `common_name` field automatically, so it's not exactly equal. a little annoying